docs/arrays.md

```diff
@@ -1,23 +1,23 @@
 # Arrays support
 
-In JDBC, arrays have always been a pain. Every time you're about to pass an
```

Comment:
Here and below: I don't think these commas should be removed. The "In JDBC" clause in the introductory, and thus must be separated with comma. That's the number 2 point from this guide:

https://www.brandeis.edu/writing-program/resources/students/academic/style-grammar/comma-rules.html

But can you change my mind?